hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -765,6 +774,7 @@ protected Object putInternal(Data key, Object value, long ttl, boolean loadFromS
         if (record == null) {
             record = createRecord(value, ttl, now);
             storage.put(key, record);
+            eventJournal.writeAddEvent(mapContainer.getObjectNamespace(), partitionId, record.getKey(), record.getValue());
```

Comment:
ok if there will be no store, this is not an issue but if we will add store to ringbuffer later, i expect it to behave as in`mapstore` case. Currently, if something cannot be stored in `mapstore`, operations fail.